modules/ROOT/pages/logging-mdc.adoc

```diff
@@ -0,0 +1,112 @@
+= MDC Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: tracing, correlation, id, logging, mdc, variables, anypoint, studio
+
+The idea behind MDC (_Mapped Diagnostic Context_) is to enrich logging to provide more context or information that could be available in the current event, to improve tracking.
+
+In the following example without MDC:
```

Comment:
By default, mule uses two MDC entries: the "processor" (to show the current event location) and the "event" (to show the event correlation id). So, if you don't use the Mule Tracing Module: